project.clj

```diff
@@ -15,6 +16,15 @@
                  [clj-yaml "0.4.0" :exclusions [org.yaml/snakeyaml]]
                  [org.yaml/snakeyaml "1.15"]
                  [commons-codec/commons-codec "1.10"]]
+  :aliases {"at" ["test-refresh"]
```

Comment:
It's fine to have things like this in private projects, but if someone outside of FC decides to use it and has `at` mapped to something else globally, lein will be complaining.